src/test/java/core/basesyntax/service/implemantation/SaveToStorageImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.implemantation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.SaveToStorageService;
+import core.basesyntax.service.strategy.TransactionBalance;
+import core.basesyntax.service.strategy.TransactionHandler;
+import core.basesyntax.service.strategy.TransactionReturn;
+import core.basesyntax.service.strategy.TransactionSell;
+import core.basesyntax.service.strategy.TransactionStrategy;
+import core.basesyntax.service.strategy.TransactionStrategyImpl;
+import core.basesyntax.service.strategy.TransactionSupply;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SaveToStorageImplTest {
+    private SaveToStorageService saveToStorageService;
+    private TransactionStrategy transactionStrategy;
+    private Map<String, TransactionHandler> handlerMap;
+    private List<String> stringList;
+
+    @Before
+    public void setUp() {
+        handlerMap = new HashMap<>();
+        handlerMap.put("b", new TransactionBalance());
+        handlerMap.put("p", new TransactionSell());
+        handlerMap.put("r", new TransactionReturn());
+        handlerMap.put("s", new TransactionSupply());
+        transactionStrategy = new TransactionStrategyImpl(handlerMap);
+        saveToStorageService = new SaveToStorageImpl(transactionStrategy);
+        stringList = new ArrayList<>();
```

Comment:
You only need it in one method so there is no need to make it a class level field